src/frontend/js/widgets/Dashboard/components/TeacherDashboardOrganizationSidebar/index.tsx

```diff
@@ -69,7 +69,7 @@ export const TeacherDashboardOrganizationSidebar = () => {
       menuLinks={links}
       header={organization.title}
       subHeader={intl.formatMessage(messages.subHeader)}
-      avatarPosition={DashboardAvatarPositionEnum.THREE_QUARTER}
+      avatarPosition={DashboardAvatarPositionEnum.CENTER}
```

Comment:
`DashboardAvatarPositionEnum.THREE_QUARTER` is now onlly used as default avatar position in [DashboardSidebar](https://github.com/openfun/richie/blob/master/src/frontend/js/widgets/Dashboard/components/DashboardSidebar/index.tsx)
Do we want to remove this `avatarPosition` prop and always use `DashboardAvatarPositionEnum.CENTER` for both teacher and learner sidebars ?

Reply:
Indeed we could remove this variant